src/common/variables.c

```diff
@@ -64,6 +65,8 @@ typedef struct dt_variables_data_t
 
   uint32_t tags_flags;
 
+  int count_versios;
```

Comment:
I thought I had already commented this ...
Is this count used ? (I've not seen it else where)
If yes, should not be count_versions ?

Reply:
d'oh you're right. I started out adding it to this struct but then realised the new variable would be so rarely used it would just be inefficient. I'll remove it.